test/custom-naming-test.js

```diff
@@ -20,12 +20,13 @@ describe('Custom delimeters for BEM naming', function() {
         block: 'b1',
         mix: { elem: 'e2' }
       }
-    ], '<div class="b1$$e1 b1$$e1@@a@@b"></div>' +
+    ], '<div class="b1$$e1 b1$$e1@@a**b"></div>' +
       '<div class="b1$$e1"></div>' +
       '<div class="b1 b1$$e2"></div>', {
       naming: {
         elem: '$$',
-        mod: '@@'
+        mod: '@@',
+        modVal: '**'
```

Comment:
@zxqfox 👍 

I implemented such API. Now naming option can be {String|Object}:

`{ naming: { mod: '>>>' } }` or `{ naming: { mod: { name: '>>>', val: '_' } }`